src/gc/env/gcenv.ee.h

```diff
@@ -79,6 +81,8 @@ class GCToEEInterface
     static void GcEnumAllocContexts(enum_alloc_context_func* fn, void* param);
 
     static void AttachCurrentThread(); // does not acquire thread store lock
```

Comment:
Can we remove AttachCurrentThread from GCToEE interface now? It does not seem to be used anymore.